### PR TITLE
fix(bb_rust): Fix/fixing dep issues

### PR DESCRIPTION
**Pull request by skukan159**
Trying to fix following errors:
They only trigger when running cargo check in other services importing bb_rust.

```
error[E0432]: unresolved import `tracing_subscriber::filter::FilterExt`
  --> /Users/filipskukan/.cargo/git/checkouts/bb_rust-059bcb4701f6f5e6/d31b5cd/src/aws_lambda_utils/mod.rs:14:5
   |
14 | use tracing_subscriber::filter::FilterExt;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^---------
   |     |                           |
   |     |                           help: a similar name exists in the module: `FilterId`
   |     no `FilterExt` in `filter`

error[E0433]: failed to resolve: could not find `enabled` in `tracing`
   --> /Users/filipskukan/.cargo/git/checkouts/bb_rust-059bcb4701f6f5e6/d31b5cd/src/aws_lambda_utils/xray_layer.rs:102:49
    |
102 | ...                   if tracing::enabled!(Level::TRACE) {
    |                                   ^^^^^^^ could not find `enabled` in `tracing`

error[E0407]: method `record_i128` is not a member of trait `Visit`
   --> /Users/filipskukan/.cargo/git/checkouts/bb_rust-059bcb4701f6f5e6/d31b5cd/src/aws_lambda_utils/tracing_fmt_layer.rs:108:5
    |
108 |       fn record_i128(&mut self, field: &tracing::field::Field, value: i128) {
    |       ^  ----------- help: there is an associated function with a similar name: `record_i64`
    |  _____|
    | |
109 | |         self.ser.serialize_field(field.name(), &value).ok();
110 | |     }
    | |_____^ not a member of trait `Visit`

error[E0407]: method `record_u128` is not a member of trait `Visit`
   --> /Users/filipskukan/.cargo/git/checkouts/bb_rust-059bcb4701f6f5e6/d31b5cd/src/aws_lambda_utils/tracing_fmt_layer.rs:112:5
    |
112 |       fn record_u128(&mut self, field: &tracing::field::Field, value: u128) {
    |       ^  ----------- help: there is an associated function with a similar name: `record_u64`
    |  _____|
    | |
113 | |         self.ser.serialize_field(field.name(), &value).ok();
114 | |     }
    | |_____^ not a member of trait `Visit`

error[E0624]: associated function `record` is private
   --> /Users/filipskukan/.cargo/git/checkouts/bb_rust-059bcb4701f6f5e6/d31b5cd/src/aws_lambda_utils/tracing_request_id_layer.rs:31:32
    |
31  |                 attrs.values().record(&mut visitor);
    |                                ^^^^^^ private associated function
    |
   ::: /Users/filipskukan/.cargo/registry/src/github.com-1ecc6299db9ec823/tracing-core-0.1.21/src/field.rs:766:5
    |
766 |     pub(crate) fn record(&self, visitor: &mut dyn Visit) {
    |     ---------------------------------------------------- private associated function defined here

error[E0599]: no method named `or` found for struct `EnvFilter` in the current scope
  --> /Users/filipskukan/.cargo/git/checkouts/bb_rust-059bcb4701f6f5e6/d31b5cd/src/aws_lambda_utils/mod.rs:55:60
   |
55 |                 .with_filter(EnvFilter::from_default_env().or(RequestIdLayer)),
   |                                                            ^^ method not found in `EnvFilter`

Some errors have detailed explanations: E0407, E0432, E0433, E0599, E0624.
For more information about an error, try `rustc --explain E0407`.
error: could not compile `bb_rust` due to 6 previous errors
```

### CODE REVIEW

**Comment by simonvandel** [comment]
I don't think this is the correct approach. What features are you giving to bb_rust in the project you are trying to import bb_rust?

**Review by martinjlowm — APPROVED** [review]

**Review by martinjlowm — COMMENTED** [review]
Hmm actually... it compiles fine without registry added.

File: Cargo.toml [hunk 1]
```diff
@@ -70,6 +70,7 @@ tracing = { version = "0.1", optional = true }
 tracing-subscriber = { version = "0.3", features = [
   "tracing-log",
   "env-filter",
+  "registry",
```
Comment (martinjlowm) [inline Cargo.toml:73]:
```suggestion
```

**Review by simonvandel — APPROVED** [review]
The current revision that is just removing the duplicate tracing-subscriber looks good to me.

**Pull request merged**